### ISSUE TITLE
theme button and javascript code button

### ISSUE THREAD

**Issue by Faizan707** [reduxjs/redux-toolkit#4766]
when cursor goes to the theme button it change into not allowed and javascript button not working it is not showing the javascript redux code

**Comment by markerikson** [comment]
I assume you're talking about the docs?

Can you include a screenshot of what you're seeing? What browser are you using?

**Comment by Faizan707** [comment]
![Image](https://github.com/user-attachments/assets/b072f00a-0047-4639-bd5c-4e5abd7a4096)
![Image](https://github.com/user-attachments/assets/fe82b055-a759-428b-a3d4-b8156a925874)

I was setting up Redux in my project and visited the official documentation to copy the required code. However, I noticed that certain interactive elements on the website, such as the JavaScript toggle button and the theme switch button, are not functioning as expected.
When I hover over these buttons, the cursor changes to a "not allowed" sign, indicating they are disabled or broken. This issue persists across multiple browsers (tested on Chrome, Edge, and Firefox). Could you please look into this and resolve the issue?

**Comment by matteo0402** [comment]
Same here on both Chrome and Edge. The search can't be opened either.

**Comment by markerikson** [comment]
Hmm. We did some updates to the repo to support React 19, and did run into some issues rebuilding our docs in the process. I bet this is related.

**Comment by markerikson** [comment]
Probably related, yeah

**Comment by dimuch** [comment]
Hey 
itis already 3 days
lets guys fix this bug
<img width="1676" alt="Image" src="https://github.com/user-attachments/assets/92736e14-4d82-4f14-b4a5-b1f734230c71" />

**Comment by markerikson** [comment]
@dimuch **demanding fixes like that is _not_ acceptable!**

We do this for free, in our spare time.  As you've already seen by the fact that I commented in this thread, we're aware of this issue.  Even if we hadn't seen this, _demanding_ that we fix this is incredibly rude. **Don't do that again**.

On top of that, this isn't even a critical library code issue!  This is just "the search input and copy code buttons don't work".

Having said that: I spent _9 hours of my free time_ yesterday working on making fixes and improvements to our docs build setup.  I have a draft PR up to fix this, and it needs a couple more pieces to be ready.

But please, have some respect to the people who spend hundreds of hours doing this for free.

**Comment by rcherny** [comment]
Can confirm on localhost that the dev docs are dramatically more fixed and working from the PR branch. Looking forward to it. Thanks for all the hard work @markerikson — it is appreciated!

**Comment by markerikson** [comment]
Okay, this should be fixed now!

The full explanation of what happened is over in the PR:

- #4774 

Loosely:

- We upgraded the RTK repo to support React 19
- That broke the client-side interactivity because the Docusaurus v2 site tool doesn't support React 19 properly
- Upgrading to Docusaurus v3 fixed the React 19 issue, but our TS snippet compiler plugin broke.  Docusaurus now expects MDX format v3 syntax, and we had a bug in our attempt to upgrade the plugin's internals to support MDX v3. That ended up deleting all code snippets from the pages (!), so we had to fix the plugin before we could complete the upgrade that fixed the search / theming / code copy button issue
